### PR TITLE
fix issue #9

### PR DESCRIPTION
**Pull request by unCleanCode**

### CODE REVIEW

File: ping.go [hunk 1]
```diff
@@ -84,6 +84,7 @@ func NewPinger(addr string) (*Pinger, error) {
 		ipv4 = false
 	}
 
+    rand.Seed(time.Now().UnixNano())
```
Comment (SuperQ) [inline ping.go]: This looks like it needs `go fmt`.

**Comment by ricardoatsouza** [comment]
This also fixes this issue (https://github.com/sparrc/go-ping/issues/14).

**Comment by sparrc** [comment]
I believe this issue was caused by previously the code was not checking the icmp id field, can you confirm this @ricardoatsouza ?

**Comment by ricardoatsouza** [comment]
@sparrc `rand.Seed` is still necessary to guarantee that `rand.Intn(0xffff)` will not collide when running two processes at the same time.  I wrote a quick explanation here: https://github.com/sparrc/go-ping/issues/14#issuecomment-436254758

**Comment by sparrc** [comment]
I'm not sure that this PR will fix the issue in all cases. Setting the random seed sets a global seed so the pingers would likely use the same seed unless one was started before the other was instantiated. I think we need to embed a random number generator with a seed into the struct.

**Comment by sparrc** [comment]
see #43

**Comment by ricardoatsouza** [comment]
@sparrc I see what you mean. While it does solve the problem for two running processes, it might still cause conflicts internally when using two pingers. Good point.

**Pull request closed without merging**